src/main/java/com/jayway/maven/plugins/android/phase04processclasses/ProguardMojo.java

```diff
@@ -429,7 +442,7 @@ private void executeProguard() throws MojoExecutionException
             IOUtils.write( commandStringBuilder, tempConfigFileOutputStream );
 
             executor.setCaptureStdOut( true );
-            commands.add( "@\"" + tempConfigFile.getAbsolutePath() + "\"" );
+            commands.add( "@" + tempConfigFile.getAbsolutePath() + "" );
```

Comment:
I have a feeling this is breaking in some other situation then... Mac or Linux or something else. This was a pain to get right ..